dockerfiles/Dockerfile.tensorrt

```diff
@@ -26,8 +26,8 @@ RUN git clone --single-branch --branch ${ONNXRUNTIME_BRANCH} --recursive ${ONNXR
 
 # Checkout appropriate TRT_VERSION and build
 RUN cd onnxruntime &&\
-    trt_version=${TRT_VERSION%.*.*} &&\
+    trt_version=${TRT_VERSION:0:3} &&\
```

Comment:
a bit more info: by default, Nvidia containers already have a TRT_VERSION environment variable, meaning the one we pass in is useless

21.07: 8.0.1.6+cuda11.3.1.005